package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "eslint-config-formidable",
   "description": "A set of default eslint configurations from formidable",
-  "version": "1.0.0",
+  "version": "0.0.0",
   "authors": "eric.baer@formidable.com, ryan.roemer@formidable.com, kyle.cesmat@formidable.com",
```

Comment:
A tag should represent the state of the repository at a release version. npm install X@3.0.0 should produce an empty diff against the repo at the 3.0.0 tag wit the exception of what is `.npmignore`d of course. There are some people who do a pre-release build then push to npm but either way the tag in the repo comes first. This should be true of not just this project but most any project.

* Prep release - bump version, update changelog
* Tag master
* `git checkout <tag>`
* 'npm publish'

How the tag is created isn't too important so long as the tag === the deploy.